run.mac.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+osname=$(uname)
```

Comment:
@briandrawert I was running ./run.ubuntu.sh on my Mac by mistake, So those lines take care of giving an error when I run the linux StochSS starter script on my Mac and vice versa

@bbbales2 , I forgot to mention because of how celery is configured, you can either run CLI or StochSS from browser and not both at the same time.